webapp/src/components/tutorial/TutorialCallout.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+
+import { Button } from "../../sui";
+
+interface TutorialCalloutProps extends React.PropsWithChildren<{}> {
+    className?: string;
+    buttonIcon?: string;
+    buttonLabel?: string;
+
+    onClick?: (visible: boolean) => void;
+}
+
+export function TutorialCallout(props: TutorialCalloutProps) {
+    const { children, className, buttonIcon, buttonLabel } = props;
+    const [ visible, setVisible ] = React.useState(false);
+
+    const captureEvent = (e: any) => {
+        e.preventDefault();
+        e.stopPropagation();
+        e.nativeEvent?.stopImmediatePropagation();
+    }
+
+    const closeCallout = (e: any) => {
```

Comment:
not for this pr, but this "click anywhere and dismiss" behavior would be a good candidate for react-common

Reply:
oh yeah definitely, i was a little torn on whether to start integrating the react-common components (buttons, etc) here, but stuck with SUI for now just for visual consistency within the webapp